wgsl/index.bs

```diff
@@ -1025,22 +1024,15 @@ variable_stmt
 ## If Statement ## {#if-statement}
 <pre class='def'>
 if_stmt
-  : IF paren_rhs_stmt body_stmt {(elseif_stmt else_stmt?) | (else_stmt premerge_stmt?)}
+  : IF paren_rhs_stmt body_stmt {(elseif_stmt else_stmt?) | else_stmt}
```

Comment:
I think this can just become `IF paren_rhs_stmt body_stmt elseif_stmt? else_stmt?` after this change

Reply:
I agree but we can do this fix-up after.